test-locales.js

```
@@ -0,0 +1,15 @@
+
+const { supportedLocalesToLanguages } = require('./packages/lib/locale.js');
+const { supportedLocales } = require('./packages/lib/locale.js');
+
+try {
+    const locales = supportedLocales();
```

ESLint

[error] Expected indentation of 1 tab but found 4 spaces.

(indent)